core/primitives/src/sharding.rs

```diff
@@ -276,8 +274,6 @@ impl ShardChunkHeaderV3 {
                 bandwidth_requests,
             })
         } else if let Some(congestion_info) = congestion_info {
```

Comment:
Change parameter `congestion_info` in function `ShardChunkHeaderV3::new()` to take parameter as `congestion_info: CongestionInfo`